### PR TITLE
Use active_storage_base64

### PR DESCRIPTION
**Pull request by santib**
Use our own gem 😄

### CODE REVIEW

**Comment by fedeagripa** [comment]
@santib I was planning to do a pr to migrate to rails 6 and add this gem, do you mind solving the conflicts or may I ?

**Comment by santib** [comment]
@fedeagripa you can do it if you want

File: Gemfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ ruby '2.4.1'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.2'
 
+gem 'active_storage_base64'
```
Comment (Alxzu) [inline Gemfile]: Why not pointing v 0.1.3?

Comment (santiagovidal) [inline Gemfile]: We won't be uploading many new versions to it, but for consistency maybe we can specify the version here.

**Pull request merged**